using_images/other_images/jenkins.adoc

```diff
@@ -514,6 +514,33 @@ an entrypoint. For more details about this, refer to the official
 https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds#Distributedbuilds-Launchslaveagentheadlessly[Jenkins
 documentation].
 
+==== Permission considerations
+
+Consider for a moment in the above ConfigMap example the `<serviceAccount>` element of the 
+Pod Template XML.  As the name implies, it is the {product-title} Service Account used for the 
+resulting Pod, where those Service Account credentials mounted into the Pod, with whatever permissions 
+are associated with the ServiceAccount, control which operations against the {product-title} master
+are allowed from the Pod.
+
+Some details to consider with regard to the Service Accounts used for Pod launched by the Kubernetes
```

Comment:
s/Some details to consider with regard to the Service Accounts used for Pod/Consider the following with service accounts used for the Pod,